products/appengine/api.yaml

```diff
@@ -440,3 +440,35 @@ objects:
             name: 'shell'
             description: |
               The format should be a shell command that can be fed to bash -c.
+  - !ruby/object:Api::Resource
```

Comment:
Since this is a strange resource, I went ahead and added an example YAML that should get you started:

```suggestion
  - !ruby/object:Api::Resource
    name: 'UrlDispatchRule'
    description: |
      Rules to match an HTTP request and dispatch that request to a service.
    base_url: 'apps/{{project}}'
    self_link: 'apps/{{project}}'
    create_url: 'apps/{{project}}'
    create_verb: :PATCH
    update_url: 'apps/{{project}}'
    update_verb: :PATCH
    identity:
      - 'domain'
      - 'path'
      - 'service'
    nested_query: !ruby/object:Api::Resource::NestedQuery
      modify_by_patch: true
      keys:
        - dispatchRules
    update_mask: true
    references: !ruby/object:Api::Resource::ReferenceLinks
      api: 'https://cloud.google.com/appengine/docs/admin-api/reference/rest/v1/apps#UrlDispatchRule'
    properties:
      - !ruby/object:Api::Type::String
        name: 'domain'
        description: |
          Domain name to match against. The wildcard "*" is supported if specified before a period: "*.".
          Defaults to matching all domains: "*".
        default_value: "*"
      - !ruby/object:Api::Type::String
        name: 'path'
        description: |
          Pathname within the host. Must start with a "/". A single "*" can be included at the end of the path.
          The sum of the lengths of the domain and path may not exceed 100 characters.
      - !ruby/object:Api::Type::String
        name: 'service'
        description: |
          Resource ID of a service in this application that should serve the
          matched request. The service must already exist.
```

Reply:
Thank you.. we my new changes, I was able to update dispatch rules and also delete. I am doing testing. Please review latest changes.